ddtrace/llmobs/_llmobs.py

```diff
@@ -157,15 +157,18 @@ def _start_service(self) -> None:
 
     def _stop_service(self) -> None:
         try:
-            self._llmobs_span_writer.stop()
-            self._llmobs_eval_metric_writer.stop()
+            self._evaluator_runner.stop()
```

Comment:
nit: maybe we want to move these out to their own try/excepts? this way, if the evaluator fails to stop/flush, we can still flush the span/eval writers. we might also be able to give better log statements in case of error

Reply:
> we can still flush the span/eval writers.

Good call out - i think we only want to explicitly called `periodic` to flush ragas traces/eval metrics from the writers, so i've moved the `periodic` calls to inside the eval runner's `stop` method.

I think the writer's themselves already have a `periodic` call registered on process exit